edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc

```diff
@@ -0,0 +1,86 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="telco-upgrade-completing-the-upgrade"]
+= Completing the telco core CNF cluster upgrade
+include::_attributes/common-attributes.adoc[]
+:context: completing-the-upgrade
+
+toc::[]
+
+Use this information to perform the cluster upgrade and monitor the upgrade through to completion.
+
+include::modules/telco-upgrade-acknowledging-the-upgrade.adoc[leveloffset=+1]
+
+include::modules/telco-upgrade-starting-the-cluster-upgrade.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-api.adoc#telco-upgrade-selecting-the-target-release_telco-upgrade-api[Selecting the target release]
+
+include::modules/telco-upgrade-monitoring-the-cluster-upgrade.adoc[leveloffset=+1]
+
+include::modules/telco-upgrade-upgrading-the-olm-operators.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc#telco-upgrade-upgrading-the-worker-nodes_completing-the-upgrade[Upgrading the worker nodes]
+
+include::modules/telco-upgrade-performing-the-second-y-stream-update.adoc[leveloffset=+2]
+
+include::modules/telco-upgrade-acknowledging-the-y-stream-release-upgrade.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://access.redhat.com/articles/6958395[Preparing to upgrade]
```

Comment:
This is a little confusing to link to a specific version's Kbase. These are only here because they needed to be released earlier than the GA release.

Technically the same exact information is in the product docs. I know you can't xref here because in 4.17 there aren't any removed. But I'd still hard link to the docs.redhat version of whatever version you're looking for

https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/updating_clusters/preparing-to-update-a-cluster#kube-api-removals_updating-cluster-prepare